cmd/sourced/compose/workdir/org.go

```diff
@@ -0,0 +1,26 @@
+package workdir
+
+import (
+	"encoding/base64"
+	"strings"
+)
+
+// InitWithOrgs initialize workdir with remote list of organizations
+func InitWithOrgs(orgs []string, token string) (string, error) {
+	workdir := base64.StdEncoding.EncodeToString([]byte(strings.Join(orgs, ",")))
```

Comment:
Shouldn't we need to sort `orgs` first?

Reply:
please take a look in the description. It's in todo for multi-org support. Currently, it works only with 1 org. There is no need to sort 1 org.

Reply:
ops, sorry I missed that. 👍